### ISSUE TITLE
update readme with list of rules and comments

### ISSUE THREAD

**Issue by SimonLab** [dwyl/dwylbot#105]
To let the users which rules is implemented and what are the action dwylbot applied to the issues/PRs we need to update the readme with 
- list of rules
- list of comments,
- list of labels added, removed
-l inks to the rule issue

**Comment by ghost** [comment]
@SimonLab if you paste in the code here for the rules I can translate to writing and update the readme

**Comment by iteles** [comment]
@markwilliamfirth I'm thinking we could spread the code knowledge across the team and as such asked @SimonLab to talk through dwylbot with @cleop this afternoon. 
Might be a good exercise for her as she delves into the code?

**Comment by Cleop** [comment]
Going to base a lot of this on: https://github.com/dwyl/dwylbot/issues/5

**Comment by SimonLab** [comment]
It might be good to double check with the actions defined for each rules in the code too

**Comment by Cleop** [comment]
@SimonLab - although I imagine some of the actions aren't yet in the code, even though they will be once the project is finished. Shall I base the list I'm currently writing on what has been done so far or what we would like to be done once the project is complete?

**Comment by ghost** [comment]
@cleop the rules have adapted since that issue so the source of truth is the code. Not all of those rules will necessarily make it in - it's best to base a readme off what's been written into the project (what's been done so far). If you wish you can add a separate section for "Upcoming features" that may change

**Comment by SimonLab** [comment]
I think on what has been done so far. Then on the next PRs for new rules we need to add the description on the readme, I should have done that from the start. I think there are in some cases some more or different actions implement ed that are not in the epic issue

**Comment by Cleop** [comment]
Great. @SimonLab @markwilliamfirth - how would you feel if I recorded these rules etc in a table format? I think it might be the easiest way to visibly read this content. Happy to move away from this if it doesn't prove effective but thought it might be worth pursuing as a starting point?

**Comment by ghost** [comment]
@Cleop table sounds like a nice idea - go with whatever format you think is best at communicating the project to the user 👍

**Comment by Cleop** [comment]
Awesome. Would you like the write up for the 'comment' that the dwylbot leaves to be verbatim or a summary?
This is using a screenshot of the comment dwylbot leaves however it's pretty small because of the width of the table and resizing the image doesn't change that because of the table dimensions.

| Category      | Rule                    | dwylbot comments                                                                                               | Labels changed |
|---------------|-------------------------|----------------------------------------------------------------------------------------------------------------|----------------|
| Pull Requests |                         |                                                                                                                |                |
|               |                         | ISSUES                                                                                                         |                |
| Issues        | Must have a description | ![image](https://user-images.githubusercontent.com/16775804/27378074-a04b55b6-566e-11e7-946a-4e0c08915dbe.png) | -              |

OR OPTION 2- copied text but no formatting of new lines etc.

| Category      | Rule                    | dwylbot comments                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Labels changed |
|---------------|-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------|
| Pull Requests |                         |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |                |
|               |                         | ISSUES                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |                |
| Issues        | Must have a description #76  | :warning: @Cleop, this issue has no description. Please add a description to help others understand the context of this issue. | -              |

Without the text after '--' I prefer option 2 because now it's a good size but also readable.

**Comment by Cleop** [comment]
| Category      | Rule                    | dwylbot comments                                                                                               | Labels changed |   |
|---------------|-------------------------|----------------------------------------------------------------------------------------------------------------|----------------|---|
| Pull Requests |                         |                                                                                                                |                |   |
|               |                         |                                                                                                                |                |   |
| Issues        | Must have a description | ![image](https://user-images.githubusercontent.com/16775804/27377591-4b09f734-566d-11e7-8791-1b43b5e2c8f1.png) | -              |   |

**Comment by ghost** [comment]
@cleop happy for you to decide - verbatim might be a good idea as this will allow us to easily review the comments and know if the wording needs to be updated. The wording/user exerience for each comment hasn't properly been reviewed yet as it's not a priority at the moment. You can omit everything under the -- if you want as it's the same on every comment (maybe have a different section for comment footer?) - whatever you think works best!

**Comment by ghost** [comment]
Closing as this was merged
